cgmanifest.json

```diff
@@ -30517,4 +30517,4 @@
     }
   ],
   "Version": 1
-}
+}
```

Comment:
Can you set your editor to "Insert final newline"? to keep the newline as last part of each file. See https://github.com/microsoft/CBL-Mariner/pull/7147